MAINTAINERS.md

```diff
@@ -6,13 +6,13 @@
 | Maintainer          | Email                             | GitHub ID                                   | Affiliation                                                                                       | Joined     |
 |---------------------|-----------------------------------|---------------------------------------------|---------------------------------------------------------------------------------------------------|------------|
 | Christian Schlotter | <christian.schlotter@daimler.com> | [chrischdi](https://github.com/chrischdi)   | Daimler TSS GmbH, [imprint](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md) | 2019-08-07 |
-| Stefan Büringer     | <stefan.bueringer@daimler.com>    | [sbueringer](https://github.com/sbueringer) | Daimler TSS GmbH, [imprint](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md) | 2019-08-07 |
 
-<!-- 
+
 ## Emeritus Maintainers
 
 | Maintainer | GitHub ID | Affiliation | Left |
 |------------|-----------|-------------|------|
+| Stefan Büringer     | <stefan.bueringer@daimler.com>    | [sbueringer](https://github.com/sbueringer) | Daimler TSS GmbH, [imprint](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md) | 2019-08-07 |
```

Comment:
I assume the left date is wrong?!

Reply:
Yeah company as well. Date is fixable, but what is the right company? :)

Reply:
I think company is correct because for the time being a maintainer you have been at Daimler TSS. However, I'll forward it to our FOSS people to take a look :-)

Reply:
Yup, thx. I updated the date in the meantime

Reply:
We haven't had this case before, but I'd say that yes, Daimler TSS is correct, because that was your affiliation at the time. Seems correct to me.